docs/framework/wcf/system-provided-bindings.md

```diff
@@ -1,86 +1,89 @@
-|Security|Specifies how the channel is secured:<br /><br /> -   None: The SOAP message is not secured and the client is not authenticated.<br />-   Transport: Security requirements are satisfied at the transport layer.<br />-   Message: Security requirements are satisfied at the message layer.<br />-   Mixed: Claims are carried in the message; integrity and confidentiality requirements are satisfied by the transport layer.|  
-|Session|Specifies whether this binding supports session contracts.|  
-|Transactions|Specifies whether transactions are enabled.|  
-|Duplex|Specifies whether duplex contracts are supported. Note that this feature requires support for Sessions in the binding.|  
-|Encoding|Specifies the wire format of the message. Allowable values include:<br /><br /> -   Text: for example UTF-8.<br />-   Binary<br />-   Message Transmission Optimization Mechanism (MTOM): A method for efficiently encoding binary XML elements within the context of a SOAP envelope.|  
-|Streaming|Specifies whether streaming is supported for incoming and outgoing messages. Use the `TransferMode` property on the binding to set the value. The allowable values include:<br /><br /> -   <xref:System.ServiceModel.TransferMode.Buffered>: The request and response messages are both buffered.<br />-   <xref:System.ServiceModel.TransferMode.Streamed>: The request and response messages are both streamed.<br />-   <xref:System.ServiceModel.TransferMode.StreamedRequest>: The request message is streamed and the response message is buffered.<br />-   <xref:System.ServiceModel.TransferMode.StreamedResponse>: The request message is buffered and the response message is streamed.|  
-  
-## See Also  
- [Endpoint Creation Overview](../../../docs/framework/wcf/endpoint-creation-overview.md)  
- [Using Bindings to Configure Services and Clients](../../../docs/framework/wcf/using-bindings-to-configure-services-and-clients.md)  
- [Basic WCF Programming](../../../docs/framework/wcf/basic-wcf-programming.md)
+> <xref:System.ServiceModel.NetHttpBinding> is a binding designed for consuming HTTP or WebSocket services and uses binary encoding by default. <xref:System.ServiceModel.NetHttpBinding> detects whether it is used with a request-reply contract or duplex contract and change its behavior to match - it uses HTTP for request-reply and WebSockets for duplex. This behavior can be overridden using the <xref:System.ServiceModel.NetHttpBinding.WebSocketTransportUsage> binding setting: Allowed - This is the default value and behaves as described above. Never - This prevents WebSockets from being used. Attempting to use a duplex contract with this setting will result in an exception. Always - This forces WebSockets to be used even for request-reply contracts. NetHttpBinding supports reliable sessions in both HTTP mode and WebSocket mode. In WebSocket mode sessions are provided by the transport.
```

Comment:
- change its behavior --> changes its behavior
- the hyphen between "match" and "it" should be a semicolon.
- the xref is broken; it's \<xref:System.ServiceModel.Channels.WebSocketTransportUsage>. But there's no Allowed value. I think the value is WhenDuplex. See https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.channels.websockettransportusage?view=netframework-4.7.2.